s3/replication/manager/src/s3replicationmanager/job_routes.py

```diff
@@ -145,16 +163,9 @@ async def get_jobs(request):
         return web.json_response(
             completed_jobs, dumps=Jobs.list_dumps, status=200)
 
-    elif 'count' in query:
-        # Return total jobs count.
-        _logger.debug('Returning all jobs count = {}'.format(
-            all_jobs_list.count()))
-        return web.json_response(
-            {'count': all_jobs_list.count()}, status=200)
-
     else:
         # return jobs that are not yet distributed.
-        _logger.debug('Returning All Jobs, count = {}'.format(
+        _logger.debug('Returning All Jobs count = {}'.format(
             all_jobs_list.count()))
```

Comment:
Please retain `,` as in real response we are returning all jobs with a count.

Reply:
one